### PR TITLE
new Balance For T1.5 Kbot

### PR DESCRIPTION
**Pull request by skymyj**

### CODE REVIEW

**Comment by skymyj** [comment]
So, new balance to propose : 
T1.5 Rocket Bot : 620 Range. 100 dmg/s. Cost 250m 2.5KE
T1 Rocket. No change. For info : 475 Range. 36dmg/S. Cost 100m 1ke
T1 Rocket1. range 575. 70dmg/S. Coest 200m 2KE
Gladiator. Decrease resources (8KE => 4.8KE). Rocket 575 => 475. Dmg 120 => 100/s. Laser 100 => 75smg/s.
Aexxec => No big changes. Decrease a little Energy. 5500 => 5200.

**Comment by skymyj** [comment]
After testing, resist to Brutus or Scar with a little less strong than veh.

**Comment by ZweisteinT** [comment]
sounds good!

**Comment by skymyj** [comment]
Increase Strenght of TLL Bot Rocket.

File: units/armcrack.lua [hunk 1]
```diff
@@ -74,7 +74,7 @@ return {
 				footprintz = 3,
 				height = 40,
 				hitdensity = 100,
-				metal = 52.0085,
```
Comment (nixtux) [inline units/armcrack.lua]: As ive said before its pointless editing wreak/heap values there autogenerated at game start

**Pull request merged**